cardano_node_tests/tests/test_kes.py

```diff
@@ -132,6 +133,63 @@ def test_expired_kes(
 
     @allure.link(helpers.get_vcs_link())
     @pytest.mark.order(6)
+    @pytest.mark.skipif(
+        not (VERSIONS.cluster_era == VERSIONS.transaction_era == VERSIONS.LAST_KNOWN_ERA),
+        reason="meant to run only with the latest cluster era and the latest transaction era",
+    )
```

Comment:
This test should be merged with `test_expired_kes`. This test needs to start cluster instance with specific configuration (same as `test_expired_kes`) and setup and teardown takes ~ 40 mins - it is very time consuming.
Until the merging is done, please disable the test in CI with
```python
    @pytest.mark.skipif(
        bool(os.environ.get("CI")),
        reason="meant to run only outside of CI",
    )
```

Reply:
Totally agree, I tried to merge but it's not straight forward, so I just created a new one to not mess with the running one.